### ISSUE TITLE
JetCourses plugin

### ISSUE THREAD

**Issue by rocketeer76** [Crocoblock/suggestions#1532]
While JetAppointment and JetBocking are good, I would prefer a more dedicated plugin for creating courses or a Learning Management System (LMS).

There are some examples on the market.
https://woocommerce.com/products/sensei/
https://thimpress.com/product/wordpress-lms-plugin-learnpress/
https://www.learndash.com

**Comment by EvgenDoman** [comment]
+1

**Comment by MarIseof** [comment]
+1

**Comment by ofmarconi** [comment]
+1

**Comment by Marcoqwerty78** [comment]
Yes super interested, also because Learnpress its not compatible with query builder

**Comment by ofmarconi** [comment]
> Yes super interested, also because Learnpress its not compatible with query builder

What would you like to query with Query Builder? I think LearnDash's CPT metafields are compatible.

I already populated a listingrid with these Query Builder queries

**Comment by Marcoqwerty78** [comment]
> > Yes super interested, also because Learnpress its not compatible with query builder
> 
> What would you like to query with Query Builder? I think LearnDash's CPT metafields are compatible.
> 
> I already populated a listingrid with these Query Builder queries

Italian? (Marconi...) btw i recorded a youtube video, i could share it privately. Basically i created a search page with two query to show on the same page Posts and Products, if Learnpress plugin is enabled the dynamic fields of the Post search give me back the image of the products. I used Dynamic Fields and Query to pass the search value.

**Comment by ofmarconi** [comment]
@Marcoqwerty78 Hey!
I'm of Italian descent, but I'm Brazilian :D

You can send it to me if you want: marconi @ localhost. agency

Ahhh I read it wrong, I'm sorry, I understood LearnDash, but from what you explained to me it seems a specific incompatibility of Learnpress, their team didn't answer how to solve it? From what I understand, he should not interfere with this consultation.

If you can test LearnDash I think you will get better results.

**Comment by Marcoqwerty78** [comment]
> 

Ahahahahah Hello Friend! cool mix ita/bra i love Brazil!!!!!

I tried to save Learnpress from a Theme that i trow  away cause a shit! now i' working only with this Croco suite that is amazing....except that i need a LMS.

Learnpress is licensed; then i would like to give him a chance. Lately i opened a ticket; but if they cant solve it probably i will change from LearnDash (as your advice).

Did you try deep this Learndash? thank you again!
